### ISSUE TITLE
Bug - Galaxy in a Galaxy on workflow search

### ISSUE THREAD

**Issue by nomadscientist** [galaxyproject/training-material#5634]
The workflowsearch widget (and import workflows in tutorial headers) is amazing, but when accessed from tutorial mode, leads to a Galaxy in a Galaxy.
Is there a way for it to first check if its being selected from tutorial mode, and if so, to instead move to the workflow link in the wider Web page, not in the tutorial mode box?

**Comment by hexylena** [comment]
yeah we have a proxy active CSS class that gets attached. someone could use it to fix this
